Source/WebCore/rendering/svg/SVGRenderSupport.cpp

```diff
@@ -141,25 +141,33 @@ LayoutRepainter::CheckForRepaint SVGRenderSupport::checkForSVGRepaintDuringLayou
     return LayoutRepainter::CheckForRepaint::Yes;
 }
 
-auto SVGRenderSupport::computeContainerBoundingBoxes(const RenderElement& container, RepaintRectCalculation repaintRectCalculation) -> ContainerBoundingBoxes
+static bool hasValidBoundingBoxForContainer(const RenderObject& object)
 {
-    ContainerBoundingBoxes result;
+    if (auto* svgShape = dynamicDowncast<LegacyRenderSVGShape>(object))
+        return svgShape && !svgShape->isRenderingDisabled();
 
-    for (CheckedRef current : childrenOfType<RenderObject>(container)) {
-        if (current->isLegacyRenderSVGHiddenContainer())
-            continue;
+    if (auto* text = dynamicDowncast<RenderSVGText>(object))
+        return text && text->isObjectBoundingBoxValid();
 
-        // Don't include elements in the union that do not render.
-        if (auto* shape = dynamicDowncast<LegacyRenderSVGShape>(current.ptr()); shape && shape->isRenderingDisabled())
-            continue;
+    if (auto* hiddenContainer = dynamicDowncast<LegacyRenderSVGContainer>(object))
+        return hiddenContainer && !hiddenContainer->isLegacyRenderSVGHiddenContainer();
```

Comment:
I don’t think our local variable name has to be this specific. Can just be a sensible word. Also, I think this is making an object that is a non-hidden container be in a local variable named hidden container.
```suggestion
    if (auto* container = dynamicDowncast<LegacyRenderSVGContainer>(object))
        return container && !container->isLegacyRenderSVGHiddenContainer();
```

Reply:
Done.